cmds/server.go

```diff
@@ -10,12 +10,12 @@ Run a fake TestTrack server for local development, backed by schema.yml files
 and nonsense.
 `
 
-var listenOn string
+var port int
 
-const defaultListenOn = "127.0.0.1:8297"
+const defaultPort = 8297
 
 func init() {
-	serverCmd.Flags().StringVarP(&listenOn, "listen", "l", defaultListenOn, "IP address and port to listen on")
+	serverCmd.Flags().IntVarP(&port, "listen", "l", defaultPort, "Port to listen on")
```

Comment:
so, can we call this `port` and `p` instead? 🙏

Reply:
😳 oops!